extensions/mongodb/src/test/resources/certs/ca-cert.pem

```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
It would be good to have a readme in this folder with steps how these certificates and keys were generated

Reply:
I used EE script ;) Hinted in https://github.com/hazelcast/hazelcast/pull/25301/commits/e74b2d6a8f2a0867acbb4fe89a6ab8e96fae08fe